src/memfd.rs

```diff
@@ -233,3 +248,27 @@ impl FromRawFd for Memfd {
         Memfd { file }
     }
 }
+
+impl AsRawFd for Memfd {
+    fn as_raw_fd(&self) -> RawFd {
+        self.file.as_raw_fd()
```

Comment:
`self.as_file()`?

Reply:
I kinda like how `self.file` shows that there is nothing weird going on behind a function call. But it is exactly equivalent, I would be fine with that too.